Small fixes

AlehYemelyanchyk opened the pull request.

olegpic committed:
Small fixes

iyurinok commented on src/by/epam/ayem/module4/model/City.java:
```diff
@@ -0,0 +1,23 @@
+public class City {
+
+    private String name;
+    private District district;
```
There is no getter for this private field. How are you going to access it. Also if the field is immutable (no setter it is better to mark it final)

iyurinok replied:
This comment applies to all models

iyurinok commented on src/by/epam/ayem/module4/service/CountryService.java:
```diff
@@ -0,0 +1,86 @@
+    private Set<District> districts;
+    private List<City> cities;
+
+    public CountryService() {
```
In general Service objects must be stateless and just take stateful objects as parameters.

iyurinok commented on src/by/epam/ayem/module4/service/CountryService.java:
```diff
@@ -0,0 +1,86 @@
+        countries.add(country);
+    }
+
+    public void countArea(Country country) {
```
All method names should start from verb. So I would rename it to `showCountryArea`

iyurinok commented on src/by/epam/ayem/module4/service/CountryService.java:
```diff
@@ -0,0 +1,86 @@
+        System.out.printf("The country area = %.2f km2.\n", area);
+    }
+
+    public void regionsAmount(Country country) {
```
Methods should begins with verb: set, get, print, show, run, do, etc...

iyurinok commented on src/by/epam/ayem/module4/service/CountryService.java:
```diff
@@ -0,0 +1,86 @@
+import java.util.List;
+import java.util.Set;
+
+public class CountryService {
```
You should have javadoc for all public classes as well as public methods

iyurinok replied:
There are no unit tests. All classes with business logic must be covered with unit tests.

AlehYemelyanchyk replied:
Got it! I'm going to add them later, once I'm done fixing the actual code in all the tasks.

iyurinok commented on src/by/epam/ayem/module4/service/CountryService.java:
```diff
@@ -0,0 +1,86 @@
+        this.countries = new HashSet<>();
+        this.regions = new HashSet<>();
+        this.districts = new HashSet<>();
+        this.cities = new ArrayList<>();
```
If Service is stateless, to what model object these fields belongs to?

olegpic committed:
1. Added getters to private fields.
2. Made the Service objects stateless, added stateful objects to its
parameters.
3. Renamed methods so their names starts from verbs. Also split some in
two methods with a proper naming regarding their purpose.
